pkg/csi_driver/controller_test.go

```diff
@@ -1398,6 +1399,90 @@ func TestCreateSnapshot(t *testing.T) {
 	}
 }
 
+func TestDeleteSnapshot(t *testing.T) {
+	backupName := "mybackup"
+	project := "test-project"
+	zone := "us-central1-c"
+	region := "us-central1"
+	instanceName := "myinstance"
+	shareName := "myshare"
+	cases := []struct {
+		name        string
+		createReq   *csi.CreateSnapshotRequest
+		deleteReq   *csi.DeleteSnapshotRequest
+		backupState string
+		expectErr   bool
+	}{
+		{
+			name: "Create singleshare snapshot and delete it",
```

Comment:
Once the Multishare CreateSnapshot test is merged in, we can add a case here for deleting multishare snapshots